site/content/3.12/aql/functions/document-object.md

```diff
@@ -76,6 +79,43 @@ FOR attributeArray IN attributesPerDocument
 
 This is an alias for [`LENGTH()`](#length).
 
+## ENTRIES()
+
+`ENTRIES(document) → pairArray`
+
+Return the top-level attributes of the `document` as an array of key-value pairs.
+
+To only return the attribute keys, see the [`ATTRIBUTES()` function](#attributes).\
+To only return the attribute values, see the [`VALUES()` function](#values).
+
+- **document** (object): an arbitrary document / object
+- returns **strArray** (array): the attributes of the input `document` as an
+  array of arrays. Each element is a nested array comprised of two strings with
```

Comment:
> Each element is a nested array comprised of **two strings**

This is technically not correct. We don't stringify the values. The data types are unchanged, true, false, null, numeric values remain what they are.

Reply:
Oops, my bad. Only the keys are obviously strings.